packages/xod-cli/src/apis.js

```diff
@@ -0,0 +1,75 @@
+import fetch from 'node-fetch';
+import { endsWith } from 'ramda';
+import * as msg from './messages';
+
+const getProto = apiSuffix =>
+  endsWith('.io', apiSuffix) || endsWith('.show', apiSuffix) ? 'https' : 'http';
```

Comment:
Shooul `my-premise.io` and `my-premise.cc` behave differently? I think you’ve meant `isAmong(['xod.io', 'xod.show'])` or even `unless(endsWith('.loc'))`